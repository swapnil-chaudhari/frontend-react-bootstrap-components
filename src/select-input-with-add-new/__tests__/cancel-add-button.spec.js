import React from 'react';
import CancelAddButton from '../cancel-add-button';
import renderShallow from 'render-shallow';

describe('<CancelAddButton>', () => {
    describe('when it renders', () => {
        const props = {
            text: 'Cancel',
            onClick: jest.fn()
        };
        let component;

        beforeAll(() => {
            component = renderShallow(<CancelAddButton { ...props } />).output;
        });

        test('renders a button with an onClick and the right text', () => {
            expect(component).toMatchSnapshot();
        });

    });

});
