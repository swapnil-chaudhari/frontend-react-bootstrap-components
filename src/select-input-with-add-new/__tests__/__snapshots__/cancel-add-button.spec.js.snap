// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CancelAddButton> when it renders renders a button with an onClick and the right text 1`] = `
<Button
  bsStyle="link"
  onClick={[MockFunction]}
>
  Cancel
</Button>
`;
